src/test/scala/is/hail/variant/GenomeReferenceSuite.scala

```diff
@@ -167,10 +171,29 @@ class GenomeReferenceSuite extends SparkSuite {
     assert(kt.annotate("""i1 = Interval(Locus(foo2)("1:100"), Locus(foo2)("1:104"))""").signature.field("i1").typ == gr.interval)
 
     GenomeReference.setDefaultReference(GenomeReference.GRCh37)
+    GenomeReference.deleteReference("foo2")
   }
 
   @Test def testContigRemap() {
     val mapping = Map("23" -> "foo")
     TestUtils.interceptFatal("do not have remapped contigs in the reference genome")(GenomeReference.GRCh37.validateContigRemap(mapping))
   }
+
+  @Test def testWriteToFile() {
+    val tmpFile = tmpDir.createTempFile("grWrite", ".json")
+
+    val gr = GenomeReference.GRCh37
+    val name = Random.alphanumeric.take(12).mkString
```

Comment:
why do you need to make a random name here? why not just use`val name2 = gr.name + "2"`

(you can then delete the import on Random too).